app/src/main/java/com/nextcloud/talk/webrtc/WebRtcBluetoothManager.java

```diff
@@ -47,20 +50,23 @@
 import android.os.Looper;
 import android.os.Process;
 import android.util.Log;
+
 import org.webrtc.ThreadUtils;
 
 import java.util.List;
 import java.util.Set;
 
-public class MagicBluetoothManager {
-    private static final String TAG = "MagicBluetoothManager";
+import androidx.core.app.ActivityCompat;
+
+public class WebRtcBluetoothManager {
+    private static final String TAG = WebRtcBluetoothManager.class.getCanonicalName();
```

Comment:
should be `getSimpleName()` instead, as `canonicalName` can easily go over the tag character limit.